src/conv/heuristics/ai_heuristics.cpp

```diff
@@ -145,6 +145,47 @@ class Model
     virtual std::vector<float> ToFeatures(const ProblemDescription& problem) const = 0;
 };
 
+class Gfx90aModel : public Model
```

Comment:
Just a minor question - should it be `final`?
I guess yes and I guess all of the other models derived from `Model` should be final as well.

Reply:
@CAHEK7 Done